sentry-ruby/lib/sentry/scope.rb

```diff
@@ -146,8 +146,13 @@ def transaction_name
     end
 
     def get_transaction
-      # transaction will always be the first in the span_recorder
-      span.span_recorder.spans.first if span
+      return unless span
+
+      if span.is_a?(Sentry::Transaction)
+        span
+      else
+        span.transaction
+      end
```

Comment:
Since `transactions` are `spans`, `Transaction.transaction` also points to the correct reference, right?

If so, we could simplify this to 

```suggestion
      span.transaction
```

Or even simplify the whole method to a single line:

```ruby
def get_transaction
  span.transaction if span
end
```